### DISCUSSION TITLE
Is the project dead?

### DISCUSSION THREAD

**Discussion by filipemd** [LibreSprite/LibreSprite#482] [General]
Last commit was from six months ago.

**Comment by Battojutsu** [comment]
I still use it.

**Comment by pegvin** [comment]
From: https://libresprite.github.io/#!/news

> The ones following the development might have noticed how our GitHub went silent in these weeks (Months by now). But that's not totally true, as we kept working behind the curtains👷
>
> Since we were facing too many problems with LibreSprite code structure, we opted for a completely rewrite of it (yes, we're that crazy) as it would take us less than cleaning the current code as we go. And changed the name, since it's a complete rewrite.
>
> So, tadaa, shake it a bit, add a good dose of [@felipemanga](https://github.com/felipemanga), and you obtain LibreSprite Dotto! Yes, LibreSprite Dotto! is a work in progress, but it already supports things like scripting, multiple windows management, skins/themes, and so on. Not bad if you ask me 🤷
>
> We'll update you from time to time about what's happening in the new [NEWS](https://libresprite.github.io/#!/news) section and, if you want to follow the development, here is the [repo](https://github.com/LibreSprite/Dotto)
